src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/__snapshots__/StructuredData.test.jsx.snap

```diff
@@ -0,0 +1,16 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StructuredData should render StructuredData correctly 1`] = `
+<html>
+  <head>
+    <script
+      type="application/ld+json"
+    >
+      {"@context":"http://schema.org","@type":"AudioObject","name":"ماښامنۍ خپرونه","description":"د بي بي سي ورلډ سروس څخه پروګرام کول","duration":"PT29M30S","thumbnailUrl":"ichef.bbci.co.uk/images/ic/$recipe/p063j1dv.jpg","uploadDate":"2020-04-23T15:30:00.000Z","embedURL":"https://polling.test.bbc.co.uk/ws/av-embeds/media/korean/externalId/id/ko?morph_env=live"}
```

Comment:
$recipe needs to be replaced with a value - (perhaps by the time the URL reaches the on demand radio page, it has already been converted, so it's worth checking... https://github.com/bbc/simorgh/blob/latest/src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/media/index.js#L48

Reply:
This is from the fixture data https://github.com/bbc/simorgh/blob/latest/data/indonesia/bbc_indonesian_radio/w172xh267fpn19l.json#L92.